definitions/region/model_native_regions/osembe_v1.0.0.yaml

```
@@ -0,0 +1,31 @@
+- OSeMBE v1.0.0:
```

change the hierarchy name and all region names as well...
```suggestion
- OSeMBE v1.0:
```